mods/cnc/weapons/smallcaliber.yaml

```diff
@@ -29,82 +29,57 @@ HighV:
 		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
 	Warhead@2Eff: CreateEffect
 		Explosions: piffs
+		TargetSearchRadius: 0
+
+HighV:
+	Inherits: ^HeavyMG
+
+Vulcan:
+	Inherits: ^HeavyMG
+	ReloadDelay: 2
+	Report: gun5.aud
+	Warhead@1Dam: SpreadDamage
+		Spread: 426
+		Damage: 100
+		Versus:
+			Wood: 25
+			Light: 100
+			Heavy: 35
 
 HeliAGGun:
+	Inherits: ^HeavyMG
 	ReloadDelay: 20
 	Burst: 2
 	BurstDelay: 0
 	Range: 4c0
 	MinRange: 0c768
-	ValidTargets: Ground
```

Comment:
The internal default is `Ground, Water`, so the only difference is that water can now be force-fired upon.